develop-docs/sdk/data-model/envelope-items.mdx

```diff
@@ -421,30 +423,53 @@ details.
 
 **Envelope Headers:**
 
-*None*
+_None_
 
 **Additional Item Headers:**
 
-*None*
+_None_
+
+### Log
+
+Item type `"log"` contains an array of log payloads encoded as JSON. This allows for multiple log payloads to be sent in a single envelope item.
 
-### Logs
+Only a single log item is allowed per envelope. The `item_count` field in the envelope item header must match the amount of logs sent, it's not optional. A `content_type` field in the envelope item header must be set to `application/vnd.sentry.items.log+json`.
 
-Item type `"log"` contains a logs payload encoded as JSON. Multiple log envelope items can be sent in a single envelope.
+It's okay to mix logs from different traces into the same log envelope item, but if you do, you MUST not attach a DSC (dynamic sampling context) to the envelope header.
 
 See the <Link to="/sdk/telemetry/logs">Logs</Link> documentation for the payload
 details.
 
+Example:
+
+```json
+{
+	"type": "log",
+	"item_count": 5,
+	"content_type": "application/vnd.sentry.items.log+json"
+}
+{
+	"items": [{..log..}, {..log..}, {..log..}, {..log..}, {..log..}]
+}
+```
+
 **Constraints:**
 
-*None*
+_None_
 
 **Envelope Headers:**
 
-*None*
+_None_
 
 **Additional Item Headers:**
 
-*None*
+`item_count`
+
+: **integer, required.** The number of log items in the envelope.
```

Comment:
```suggestion
: **integer, required.** The number of log entries in the item.
```

It's just scoped to the item, unfortunately naming clashes here "log item" and "envelope item"

Reply:
Naming is hard - good catch, will fix.